### PR TITLE
Move CommandListPanel to HelpWindow to optimize focus

### PR DESCRIPTION
**Pull request by gauravuj**
## Overview
This PR addresses a previously expressed need to enhance user focus by relocating the `CommandListPanel` from its original position to the `HelpWindow`. This strategic move aims to spotlight the primary feature, `BookingList`, by minimizing distractions and optimizing the user interface layout. Additionally, we have improved the functionality of the `Take me to UG` URL, enabling it to open directly in the user's default browser, thereby enhancing the user experience.

## Changes
- **CommandListPanel Relocation**: The `CommandListPanel` has been moved to the `HelpWindow` to streamline the user interface and focus attention on the `BookingList`.
- **Direct Link Opening**: Updated the `Take me to UG` **URL functionality**. It now opens the link directly in the user's default browser, as opposed to the previous behavior of merely copying the URL to the clipboard.

## Impact
- **Enhanced User Focus**: By moving the `CommandListPanel` to the `HelpWindow`, users can now concentrate more on the main feature, `BookingList`, improving the overall usability and experience.
- **Improved Accessibility**: The direct opening of the `Take me to UG` link in the default browser simplifies the process for users, making the user guide more accessible and user-friendly.

### Help Window
Screenshot 2024-03-28 at 2 56 50 PM

### Main Window
Screenshot 2024-03-28 at 2 57 36 PM

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -90,13 +110,23 @@ public void focus() {
...
+        try {
+            URI uri = new URI(USERGUIDE_URL);
+            if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+                Desktop.getDesktop().browse(uri);
+            } else {
+                System.err.println("Browsing not supported!");
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```
Comment (cyqjoseph) [inline src/main/java/seedu/address/ui/HelpWindow.java:129]: Nice!

File: src/main/resources/view/stylesheets/DarkTheme.css [hunk 2]
```diff
@@ -325,4 +325,5 @@
     -fx-border-radius: 15;
     -fx-background-radius: 15;
     -fx-font-size: 12;
-}
\ No newline at end of file
+}
```
Comment (cyqjoseph) [inline src/main/resources/view/stylesheets/DarkTheme.css:328]: Thanks for adding these!

**Review by cyqjoseph — APPROVED** [review]
LGTM! Great fixes!

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (ChrysanthemumT) [inline src/main/java/seedu/address/ui/HelpWindow.java:129]: whats going on here?

**Pull request merged**